test/unit/dialog-controller.spec.js

```diff
@@ -16,32 +14,38 @@ describe('the Dialog Controller', function () {
     });
   }
 
-  beforeEach(function () {
-    new Container().makeGlobal();
-    settings = { name: 'Test' };
-    dialogController = new DialogController(renderer, settings, Function.prototype, Function.prototype);
+  function createDialogController(settings = {}) {
+    const dialogController = new DialogController(renderer, settings, Function.prototype, Function.prototype);
     dialogController.viewModel = {};
     dialogController.controller = {
       unbind() { }
     };
     dialogController.renderer.hideDialog = Function.prototype;
+    return dialogController;
+  }
+
+  beforeEach(function () {
+    this.dialogController = createDialogController();
```

Comment:
Gotcha, typically we've been using it like this - 

```
import {DialogController} from 'src/dialog-controller';
import {Container} from 'aurelia-dependency-injection';
describe('ModuleName', () => {
  let dialogController;
  let container;
  beforeEach(() => {
    container = new Container();
    dialogController = container.get(DialogController);
  });
});
```

which is basically the same as what you have just a slight style difference.  It works either way so no biggie, just hadn't seen it that way before.